mobile/src/screens/meditation/meditation.tsx

```diff
@@ -1,55 +1,16 @@
 import React from 'react';
-import TrackPlayer, {
-  Event,
-  State,
-  useTrackPlayerEvents,
-} from 'react-native-track-player';
 
 import imagePlaceholder from '#assets/img/card-image-placeholder.png';
 import meditationBackground from '#assets/img/meditation-background.png';
-import { Image, Text, View } from '#libs/components/components';
-import { useAppRoute, useEffect, useState } from '#libs/hooks/hooks';
+import { Image, PLayer, Text, View } from '#libs/components/components';
```

Comment:
Is it supposed to be PLayer or just typo?

Reply:
it's typo, thank you)